azure-iot-device/samples/legacy-samples/send_telemetry.py

```diff
@@ -17,8 +17,8 @@
 # Connect the client.
 device_client.connect()
 
-# send 5 messages with a 1 second pause between each message
-for i in range(1, 6):
+# send 2 messages with 2 system properties & 1 custom property with a 1 second pause between each message
```

Comment:
We definitely need E2E tests for these scenarios (and probably unit tests to make sure the topics are built correctly), but do we need this in a sample?

Reply:
I also agree that this is probably more suited to an E2E test, and perhaps the sample should stay the same.

---
In reply to: [308970700](https://github.com/Azure/azure-iot-sdk-python-preview/pull/153#discussion_r308970700) [](ancestors = 308970700)

Reply:
I think as long as there are no E2E tests let the sample be as it is

---
In reply to: [308971106](https://github.com/Azure/azure-iot-sdk-python-preview/pull/153#discussion_r308971106) [](ancestors = 308971106,308970700)